fabtests/common/shared.c

```diff
@@ -1430,7 +1430,8 @@ int ft_enable_ep_recv(void)
 
 	if (!ft_check_opts(FT_OPT_SKIP_MSG_ALLOC) &&
 	    (fi->caps & (FI_MSG | FI_TAGGED))) {
-		/* Initial receive will get remote address for unconnected EPs */
+		/* Initial receive will get remote address for unconnected EPs*/
```

Comment:
I don't think the first receive always get remote address. The address can be exchanged with OOB. The first rx just matches whatever the first tx

Reply:
Agree. This just preposts a receive. It doesn't necessarily match the send posted in ft_finalize_ep. IMO, these comments are inaccurate and unnecessary.

Reply:
@shijin-aws @j-xiong do you want me to just delete the comment on line 1433? Or do you also not like the new comments I added?

Reply:
Closed PR

Reply:
I would just close it, yes